packages/legend-application-query/src/stores/QueryTextEditorState.ts

```diff
@@ -194,6 +194,11 @@ export class QueryTextEditorState extends LambdaEditorState {
         );
       } else {
         this.queryBuilderState.initialize(this.rawLambdaState.lambda);
+        yield Promise.all([
```

Comment:
See my comment in `QueryBuilderChangeDetectionState`, we should only need to do:
`await queryBuilderState.changeDetectionState.start();` here